api/src/test/java/org/openmrs/module/registrationapp/converter/ExtensionConfigGeneratorTest.java

```diff
@@ -0,0 +1,37 @@
+package org.openmrs.module.registrationapp.converter;
+
+import java.io.InputStream;
+import java.util.List;
+import org.junit.Test;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.codehaus.jackson.type.TypeReference;
+import org.openmrs.module.appframework.domain.AppDescriptor;
+import org.openmrs.module.appframework.domain.Extension;
+import org.openmrs.module.appframework.domain.AppDescriptor;
+import org.springframework.core.io.support.PathMatchingResourcePatternResolver;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertEquals;
+
+public class ExtensionConfigGeneratorTest {
+
+	private PathMatchingResourcePatternResolver resourceResolver = new PathMatchingResourcePatternResolver();
+	
+    @Test
+    public void generateShouldTranslateAppConfigToSummayConfig() throws Exception{
```

Comment:
Let's use the usual convention → `generate_shouldGenerateRegSummaryFromRegApp`.

Reply:
Thank you @mks-d , I had an issue via this some time back with core. I have considered this.